affiliates/estonia.swagger.json

```diff
@@ -206,6 +206,7 @@
     "schemas" : {
       "EmployerAddress" : {
         "type" : "object",
+        "description" : "Your company address",
```

Comment:
this should say something in the lines of if you need business loan, then Your company address, otherwise company address you work in(for future references)

Reply:
changed